common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -141,6 +141,11 @@ class DB(contextWrapper: ContextWrapper) {
     contextWrapper.dbContext.run(action)
   }
 
+  val curatedStatuses = List(RecipeStatusVerified, RecipeStatusCurated, RecipeStatusFinalised, RecipeStatusPendingCuration, RecipeStatusPendingVerification, RecipeStatusPendingFinalisation).map(_.name)
```

Comment:
This list surprises me. Why do we consider pending recipes ready for publication?